packages/api/src/clickhouse/__tests__/clickhouse.test.ts

```diff
@@ -42,4 +43,29 @@ describe('clickhouse', () => {
     expect(clickhouse.client.insert).toHaveBeenCalledTimes(2);
     expect.assertions(2);
   });
+
+  it('getMetricsChart', async () => {
+    jest
+      .spyOn(clickhouse.client, 'query')
+      .mockResolvedValueOnce({ json: () => Promise.resolve({}) } as any);
+
+    await clickhouse.getMetricsChart({
+      aggFn: clickhouse.AggFn.AvgRate,
+      dataType: 'Sum',
+      endTime: Date.now(),
+      granularity: clickhouse.Granularity.OneHour,
+      name: 'test',
+      q: '',
+      startTime: Date.now() - 1000 * 60 * 60 * 24,
+      teamId: 'test',
+    });
+
+    expect(clickhouse.client.query).toHaveBeenCalledTimes(1);
+    expect(clickhouse.client.query).toHaveBeenCalledWith(
+      expect.objectContaining({
```

Comment:
we can also use an snapshot/inline snapshot if that's better. Though depending on the intent I imagine this is good enough

Reply:
Yeah the snapshot would change often since the query includes a date, a match was cleaner I think. Unless I'm missing something